Daily-Weather/Controller/HomeViewController.swift

```diff
@@ -87,21 +93,88 @@ extension HomeViewController : CLLocationManagerDelegate {
 
 extension HomeViewController : UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return numberOfRowHourly
+        if section == 0 {
+            numberOfRow = 1
+            return numberOfRow
+        } else if section == 1 {
+            numberOfRow = dailyModels.count
+            return numberOfRow
+        } else if section == 2 {
+            numberOfRow = 1
+            return numberOfRow
+        } else {
+            numberOfRow = 1
+            return numberOfRow
+        }
```

Comment:
```suggestion
        return section == 1 ? dailyModels.count : 1
        }
```
biến numberOfRow không cần thì bỏ luôn đi em